test/evm_script.js

```diff
@@ -279,5 +290,29 @@ contract('EVM Script', accounts => {
                 })
             })
         })
+
+        context('adding mock dying executor', () => {
+            let dyingExecutor = null
+
+            beforeEach(async () => {
+                dyingExecutor = await MockDyingScriptExecutor.new()
+                await reg.addScriptExecutor(dyingExecutor.address, { from: boss })
+            })
+
+            it('registers new executor', async () => {
+                assert.equal(await reg.getScriptExecutor('0x00000004'), dyingExecutor.address)
+            })
+
+            it('executes mock executor', async () => {
+                await executor.execute('0x00000004')
```

Comment:
Maybe this could check that the result is actually `bytes(1)`?

Reply:
Executor.execute doesn't return anything, it would require some changes

Reply:
Executor.execute doesn't return anything, it would require some changes

Reply:
Ah damn, ok. Let's not worry about it then.